services/homepage/src/components/CommonNavBar.js

```diff
@@ -9,7 +9,7 @@ class CommonNavBar extends React.Component {
         <div className={'upArrow'}>
         </div>
         {/* eslint-disable-next-line */}
-        <a id={this.props.id} className="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{this.props.navTitle}</a>
+        <button type="button" id={this.props.id} className="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{this.props.navTitle}</button>
```

Comment:
When anchor tag is used you can't open the dropdown by pressing `Enter`. It's only possible with `<button type="button">`.